actions/class.ItemImport.php

```diff
@@ -19,6 +19,9 @@
  *               2008-2010 (update and modification) Deutsche Institut für Internationale Pädagogische Forschung (under the project TAO-TRANSFER);
  *               2009-2012 (update and modification) Public Research Centre Henri Tudor (under the project TAO-SUSTAIN & TAO-DEV);
  *               2012-2021 (update and modification) Open Assessment Technologies SA;
+ *
+ * @phpcs:disable PSR1.Classes.ClassDeclaration
+ * @phpcs:disable Squiz.Classes.ValidClassName.NotCamelCaps
```

Comment:
This disables the erorrs related with the class name not being in PascalCase and not being inside a namespace.